### PR TITLE
[ADD] subscription_action

### PR DESCRIPTION
**Pull request by hbrunn**
Actions for recurring documents
===============================

This module was written to extend the functionality of the `subscription`
module in order to allow you to run a server action on newly created documents.

Configuration
=============

To configure this module, you need to:

- go to some recurring document
- select or create a server action with the same base model as your recurring document

Note that server actions already support calling multiple actions themselves, so if you need to have multiple actions, first define one umbrella action that calls the others.

### CODE REVIEW

File: subscription_action/README.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+#. select or create a server action with the same base model as your recurring document
+
+Note that server actions already support calling multiple actions themselves, so if you need to have multiple actions, first define one umbrella action that calls the others.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+    :alt: Try me on Runbot
+    :target: https://runbot.odoo-community.org/runbot/149/8.0
```
Comment (cubells) [inline subscription_action/README.rst]: It should be 9.0

**Review by cubells — CHANGES_REQUESTED** [review]

**Review by gurneyalex — APPROVED** [review]
ok for me once the small issue in the README.rst file is fixed.

File: subscription_action/README.rst (hunk 1 above)
Comment (hbrunn) [reply subscription_action/README.rst]: @cubells thanks, done & rebased

**Review by cubells — APPROVED** [review]
:+1: 

All is done.

Thanks @hbrunn

**Review by lasley — APPROVED** [review]

**Pull request merged**